src/web/shared/Contours.ts

```diff
@@ -27,7 +27,7 @@ export function geoJSONToDepthContourFeatures(projection: ProjectionLike, geojso
     return features;
 }
 
-export function geoJSONToFeatures(projection: ProjectionLike, geojson: object) {
+export function geoJSONToFeatures(projection: ProjectionLike, geojson: any) {
```

Comment:
Since this function was modified and does not have a function comment yet we should add one.